x-pack/plugins/observability/public/components/alert_search_bar/alert_search_bar_with_url_sync.tsx

```diff
@@ -27,7 +26,7 @@ function AlertSearchbarWithUrlSync(props: AlertSearchBarWithUrlSyncProps) {
       },
     },
     triggersActionsUi: { getAlertsSearchBar: AlertsSearchBar },
-  } = useKibana<ObservabilityAppServices>().services;
+  } = useKibana().services;
```

Comment:
`x-pack/plugins/observability/public/utils/kibana_react.ts` already passes in the plugin services type into the generic, so no need to re-add it here.